internal/server/stat_series.go

```diff
@@ -166,6 +162,16 @@ func (s *Server) GetStats(ctx context.Context, in *pb.GetStatsRequest) (
 	if err != nil {
 		return nil, err
 	}
+
+	latency := time.Since(ts).Seconds()
+	if latency > 30 {
+		log.Printf(
+			"GetStats() took %f seconds with: placeDcids: %s, statsVarDcid: %v",
+			latency,
+			placeDcids,
+			statsVarDcid,
+		)
+	}
```

Comment:
Is it worth putting this as a defer function so it will run no matter wherever else we return from ?

Reply:
That's a very good idea!